core/src/main/java/com/graphhopper/util/PathSimplification.java

```diff
@@ -98,6 +98,16 @@ public PointList simplify() {
                     if (removed > 0) {
                         for (int i = 0; i < toSimplify.size(); i++) {
                             reduceLength(toSimplify.get(i), offset[i], startIntervals[i], endIntervals[i] - removed);
+                            /*
```

Comment:
As you can see, the tests succeed even if we don't do this. Also the result on the map with debug=true looks the same to me in both cases, so this seems unnecessary. As described in my other comment and I would remove it before we merge this. I just left it there so you can compare it.